extension/src/counter_agg.rs

```diff
@@ -8,6 +8,7 @@ use crate::{
         AccessorExtrapolatedRate, AccessorIdeltaLeft, AccessorIdeltaRight, AccessorIntercept,
         AccessorIrateLeft, AccessorIrateRight, AccessorNumChanges, AccessorNumElements,
         AccessorNumResets, AccessorRate, AccessorSlope, AccessorTimeDelta, AccessorWithBounds,
+        AccessorFirstTime, AccessorFirstVal, AccessorLastTime, AccessorLastVal,
```

Comment:
Oh, can you restore the sorting here?

Reply:
Was waiting on the gauge agg conversation before I pushed more changes, but they're fixed now :)